src/Domain/ElementId.cpp

```diff
@@ -15,6 +15,9 @@
 #include "Utilities/MakeArray.hpp"
 #include "Utilities/StdHelpers.hpp"  // IWYU pragma: keep
 
+template <size_t VolumeDim>
```

Comment:
why is this needed?

Reply:
This is the definition of the member variable and allows it to be linked to from other libraries. C++ doesn't implicitly define `static constexpr` member variables